openslides/core/management/commands/insecurechangepassword.py

```diff
@@ -0,0 +1,28 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from openslides.users.models import User
+
+
+class Command(BaseCommand):
+    """
+    Command to change OpenSlides config values.
+    """
+    help = 'Changes OpenSlides config values.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'username',
+            help='The name of the user to set the password for'
+        )
+        parser.add_argument(
+            'password',
+            help='The new password of the user'
+        )
+
+    def handle(self, *args, **options):
+        try:
+            user = User.objects.get(username=options['username'])
```

Comment:
Please let except statement immediately follow the line which may raise the exception. The rest of the code belongs to the else block or is outside the whole try-except-else block.